components/content-service/pkg/git/git.go

```diff
@@ -197,7 +200,12 @@ func (c *Client) GitWithOutput(ctx context.Context, ignoreErr *string, subcomman
 
 	span.LogKV("args", fullArgs)
 
-	cmd := exec.Command("git", fullArgs...)
+	cmdName := "git"
+	if c.RunAsGitpodUser {
+		cmdName = "sudo"
```

Comment:
`sudo` is required?

Reply:
yeah. that is the whole point here, and assumes that this runs as root (which it is from ring0).
so it runs `sudo -u gitpod git ...`

Reply:
At this code phase, is the permission `root`?

If yes, I am thinking using `runuser` is more suitable or not. 🤔 
https://prasadlinuxblog.wordpress.com/2012/09/04/392/

Reply:
Is it safe from a security viewpoint?

Reply:
@jenting I am not sure if there is any meaningful difference in our case for sudo vs runuser. WDYT?

@utam0k I don't think so. That code was already running as root (supervisor) prior to this changes.

Reply:
Agree with you.

Reply:
I just want to confirm with you about it. :100: